test/SlowTests/Sharding/Cluster/ReshardingTests.cs

```diff
@@ -1103,11 +1103,11 @@ public async Task ShouldHaveValidBucketStatsAfterBucketMigration()
                 {
                     var stats = ShardedDocumentsStorage.GetBucketStatisticsFor(ctx, bucket);
                     Assert.Equal(bucket, stats.Bucket);
-                    Assert.Equal(7706, stats.Size);
+                    Assert.Equal(9147, stats.Size);
```

Comment:
this is because of the change vector size ?

Reply:
no
it's because now we have in bucket 2 timeseries segments (with one entry each)
and 2 more revisions